File: medplum-java/medplum-model/src/main/java/com/fhir/ValueSet_Contains.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.Boolean;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableValueSet_Contains.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ValueSet_Contains {
  Optional<List<ValueSet_Contains>> contains();

  Optional<List<Extension>> modifierExtension();

  Optional<String> display();

  Optional<List<Extension>> extension();

  @JsonProperty("abstract")
  Optional<Boolean> _abstract();

  Optional<code> code();

  Optional<List<ValueSet_Designation>> designation();

  Optional<String> id();

  Optional<String> version();

  Optional<Boolean> inactive();

  Optional<uri> system();

  static ImmutableValueSet_Contains.Builder builder() {
    return ImmutableValueSet_Contains.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/ValueSet_Contains.ts
import { code } from "./code";
import { Extension } from "./Extension";
import { uri } from "./uri";
import { ValueSet_Designation } from "./ValueSet_Designation";

export interface ValueSet_Contains {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    system: uri | null | undefined;
    abstract: boolean | null | undefined;
    inactive: boolean | null | undefined;
    version: string | null | undefined;
    code: code | null | undefined;
    display: string | null | undefined;
    designation: ValueSet_Designation[] | null | undefined;
    contains: ValueSet_Contains[] | null | undefined;
}
